presets/4.3/other/Tehllama_5in_1900-2250KV_FreedomSpec_3S-5S-6S_Race_TUNE_OTHER.txt

```diff
@@ -0,0 +1,282 @@
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/235
+
+# FORCE_OPTIONS_REVIEW: TRUE
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# Fallback if dynamic idle does not function or is deselected
+set dshot_idle_value = 440
+
+
+#$ OPTION BEGIN (CHECKED): Apply 6S Auto-Select Tune to Profile 1
+profile 0
+# profile 0 - 6S
+set iterm_rotation = ON
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set p_pitch = 40
+set i_pitch = 60
+set d_pitch = 27
+set f_pitch = 137
+set p_roll = 36
+set i_roll = 55
+set d_roll = 25
+set f_roll = 124
+set p_yaw = 36
+set i_yaw = 55
+set f_yaw = 124
+set d_min_roll = 19
+set d_min_pitch = 21
+set d_max_advance = 0
+set auto_profile_cell_count = 6
+set feedforward_boost = 5
+set throttle_boost = 2
+
+set simplified_master_multiplier = 65
+set simplified_i_gain = 85
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 160
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+simplified_tuning apply
+
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Apply 5S Auto-Select Tune to Profile 2
+profile 1
+# profile 1
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set i_pitch = 71
+set d_pitch = 33
+set f_pitch = 161
+set p_roll = 44
+set i_roll = 67
+set d_roll = 31
+set f_roll = 153
+set p_yaw = 44
+set i_yaw = 67
+set f_yaw = 153
+set d_min_roll = 23
+set d_min_pitch = 25
+set d_max_advance = 0
+set auto_profile_cell_count = 5
+set feedforward_boost = 12
+set throttle_boost = 7
+# set dyn_idle_min_rpm = 24
+set dyn_idle_p_gain = 42
+set dyn_idle_i_gain = 42
+set dyn_idle_d_gain = 42
+set simplified_master_multiplier = 80
+set simplified_i_gain = 85
+set simplified_pi_gain = 125
+set simplified_feedforward_gain = 160
+set simplified_pitch_d_gain = 105
+simplified_tuning apply
+
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Apply 3S Freedom Spec Tune on Auto-Select to Profile 3
+profile 2
+# profile 2
+set dterm_lpf1_dyn_min_hz = 100
+set dterm_lpf1_dyn_max_hz = 222
+set dterm_lpf1_static_hz = 0
+set dterm_lpf2_static_hz = 175
+set iterm_rotation = ON
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set p_pitch = 83
+set i_pitch = 126
+set d_pitch = 65
+set f_pitch = 240
+set p_roll = 75
+set i_roll = 114
+set d_roll = 57
+set f_roll = 218
+set p_yaw = 75
+set i_yaw = 114
+set f_yaw = 218
+set d_min_roll = 44
+set d_min_pitch = 51
+set d_max_advance = 0
+set auto_profile_cell_count = 3
+set feedforward_boost = 18
+set throttle_boost = 12
+
+set simplified_master_multiplier = 135
+set simplified_i_gain = 85
+set simplified_d_gain = 110
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 135
+set simplified_pitch_pi_gain = 110
+simplified_tuning apply
+
+#$ OPTION END
+
+
+#$ OPTION BEGIN (CHECKED): Set VBat Sag Compensation for ALL Profiles
```

Comment:
if that's for the real freedom-spec, then you might want to remove sag compensation for it.
People really need everything they have at the beginning of the race

But up to the author.

Reply:
Yeah, will uncheck. 
I end up needing some VBatSagComp, or else I just pancake every single race run after a really good start.